checkov/common/bridgecrew/integration_features/features/custom_policies_integration.py

```diff
@@ -103,7 +103,8 @@ def _convert_raw_check(policy: dict[str, Any]) -> dict[str, Any]:
             'id': policy['id'],
             'name': policy['title'],
             'category': policy['category'],
-            'frameworks': policy.get('frameworks', [])
+            'frameworks': policy.get('frameworks', []),
+            'scope': {'provider': policy.get('provider', '').lower()}
```

Comment:
just writing here as I have no place else - can you add a description to your PR and explain what has changed?
I can say from my standpoint - this PR without an explanation doesn't make it clear how did we change the behavior